### ISSUE TITLE
Github dependent system

### ISSUE THREAD

**Issue by ron-popov** [geofs-plugins/plugin-manager-V2#2]
A system that will host the the our development progress on one branch and another branch for production ready code.  

Our tamper monkey plugin will fetch the code from our release branch and run it.  
Every time we will push a change to the release branch our plugin will update itself and run the newer code.  
By using this method we will only have one server (Github) and won't have to pay for hosting, in addition it will be easier to manage code , and pushing code to release will be as easy as merging two branches.  

Long story short, can i delete the server folder in our repo ?

**Comment by yotam180** [comment]
You can delete it right after we have a working prototype for the GitHub based system.

**Comment by ron-popov** [comment]
Should work now, please try it yourself.  
It worked perfect for me and even handles failures.

**Comment by yotam180** [comment]
I will dig into it tomorrow, or after the arrival back from Polska.

**Comment by ron-popov** [comment]
Implemented and in release now.
